lib/slavery.rb

```diff
@@ -12,14 +12,19 @@
 module Slavery
   class << self
     attr_accessor :disabled
-    attr_writer :spec_key
+    attr_accessor :spec_key
 
-    def spec_key
-      @spec_key ||= "#{ActiveRecord::ConnectionHandling::RAILS_ENV.call}_slave"
+    def spec_key_for(target = nil)
+      spec = spec_key if target.nil? || target.to_s == "slave" # Support for Slavery.spec_key= 
+      spec || "#{ActiveRecord::ConnectionHandling::RAILS_ENV.call}_#{target}"
```

Comment:
It would load `development_two` instead of `development_slave_two`, no?

Reply:
target here is `slave_two` or `slave`. To retain your existing behavior of setting a global `Slavery.spec_key=`, if the target is `slave` and there is a `spec_key` set, then continue to use that global.